common/tests/integration_tests.rs

```diff
@@ -1,55 +1,14 @@
 use light_client::*;
 use merkle_tree::*;
-use reserved::ReservedState;
 use simperby_common::{verify::CommitSequenceVerifier, *};
+use simperby_test_suite::*;
 
 #[test]
 fn basic1() {
     let n = 10;
```

Comment:
```suggestion
    let member_number = 10;
```
or just 10 at line 9

I had to look up the declaration of `generate_standard_genesis` to find the meaning of `n`